src/app/views/customers/batch-upload/batch-upload.controller.coffee

```diff
@@ -1,7 +1,16 @@
-@App.controller 'BatchUploadController', ($window) ->
+@App.controller 'BatchUploadController', ($window, $injector, MnoeCurrentUser) ->
   'ngInject'
   vm = this
 
+  MnoeCurrentUser.getUser().then( ->
+    vm.user = MnoeCurrentUser.user
```

Comment:
You don't have to add it to `vm` as you don't need it in the view, you could just use `user`